lib/src/utils/queries.dart

```diff
@@ -11,3 +11,32 @@ const String getCollectiblesByOwnerQuery = r'''
       }
   }
 ''';
+
+const String getAccountByAddress = r'''
```

Comment:
Here also I think the previous name of the query variable was more understandable, but maybe I'm missing something